src/frontend/src/pages/index.js

```diff
@@ -6,25 +6,28 @@ import Posts from '../components/Posts';
 import ScrollToTop from '../components/ScrollToTop';
 import SEO from '../components/SEO';
 import useSiteMetaData from '../hooks/use-site-metadata';
+import CustomizedSnackBar from '../components/SnackBar';
 
 export default function IndexPage() {
   const [numPages, setNumPages] = useState(1);
   const [posts, setPosts] = useState([]);
+  const [initNumPosts, setInitNumPosts] = useState(0);
+  const [currentNumPosts, setCurrentNumPosts] = useState(0);
   const { telescopeUrl } = useSiteMetaData();
 
   useEffect(() => {
     async function getPosts(pageNum = 1) {
       try {
-        const res = await fetch(`${telescopeUrl}/posts?page=${pageNum}`);
-
+        const res = await fetch(`https://dev.telescope.cdot.systems/posts?page=${pageNum}`);
+        // const res = await fetch(`${telescopeUrl}/posts?page=${pageNum}`);
```

Comment:
I know that you're doing this here only for testing changes, but I think you can change `API_URL` in our `.env` so you don't have to commit these changes.

Reply:
Agree, this needs to use the telescopeUrl

Reply:
Fixed